Resfeber/Helpers/UITextField+Extension.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UITextField+Extension.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 12/5/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+extension UITextField {
+    
+    func setInputViewDatePicker(target: Any, selector: Selector) {
```

Comment:
I like that you implemented this in a UITextField extension so that is doesn't bog down our view controllers